factcast-site/documentation-docsy/content/en/UI/Setup/properties.md

```diff
@@ -9,6 +9,7 @@ description: Properties you want to use to configure FactCast UI.
 
 This setting should be set in every instance that factcast-server-ui is part of.
 
-| Property              | Description                                                                                                            | Default |
-| --------------------- | :--------------------------------------------------------------------------------------------------------------------- | :------ |
-| vaadin.productionMode | Should be set to true, otherwise vaadin tries to generate a dev bundle which is not necessary, and probably will fail. | false   |
+| Property                 | Description                                                                                                                                 | Default             |
+| ------------------------ | :------------------------------------------------------------------------------------------------------------------------------------------ | :------------------ |
+| vaadin.productionMode    | Should be set to true, otherwise vaadin tries to generate a dev bundle which is not necessary, and probably will fail.                      | false               |
+| factcast.ui.reports.path | The path under which reports are stored if no external ReportStore is configured. Set to a location within `/tmp` for readonly filesystems. | factcast-ui/reports |
```

Comment:
```suggestion
| factcast.ui.reports.path | The path under which reports are stored if no external ReportStore is configured. | /tmp/factcast-ui/reports |
```
The readonly aspect is probably internal, you could argue with file-system permission as well. I suggest defaulting to /tmp/. Would need to be updated in the code, of course

Reply:
factcast.ui.report.store.s3 needs to be mentioned here as well.